waspc/cli/exe/Main.hs

```diff
@@ -73,10 +73,13 @@ main = withUtf8 . (`E.catch` handleInternalErrors) $ do
   -- to be more robust than trying to only check for commands that require it.
   -- See https://github.com/wasp-lang/wasp/issues/1134#issuecomment-1554065668
   NodeVersion.getAndCheckNodeVersion >>= \case
-    Left errorMsg -> do
+    NodeVersion.VersionCheckFail errorMsg -> do
       cliSendMessage $ Message.Failure "Node requirement not met" errorMsg
       exitFailure
-    Right _ -> pure ()
+    NodeVersion.VersionCheckSuccess maybeWarning _version ->
+      case maybeWarning of
+        Just warnMsg -> cliSendMessage $ Message.Warning "Node version warning" warnMsg
+        Nothing -> pure ()
```

Comment:
How can I trigger this warning? I couldn't do it. I tried:
- Setting the version inside the `Node/Version` file to 18.0.0.
- Starting the project with Node 20

Reply:
Closing this one as other comment covers it.